components/PlaySound/PlayButton.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import Image from "next/image";
+import playing from "@/public/images/playing.gif";
+
+export default function PlayButton({ handleIsPlaying, audioPlaying }) {
+  return (
+    <StyledPlayButton
+      type="button"
+      onClick={handleIsPlaying}
+      audioPlaying={audioPlaying}
+    >
+      {audioPlaying ? (
+        <Image width={50} height={16} src={playing} alt={"playing audio"} />
+      ) : (
+        <p>turn sound on</p>
+      )}
+    </StyledPlayButton>
+  );
+}
+
+const StyledPlayButton = styled.button`
+  border: none;
+  padding-inline: 1rem;
+  padding-block: 0.1rem;
+  background-color: transparent;
+  cursor: pointer;
+
+  p {
+    color: var(--color-main-alt);
+    font-size: 0.9rem;
+    text-align: center;
+  }
+`;
```

Comment:
you should put this in a separate file. Also think about if you really need the nesting here. It might be clearer what is going on, if you style the p element itself